protostar.py

```diff
@@ -11,4 +12,7 @@
 cmd_remove_parser = root_subparsers.add_parser("remove")
 cmd_remove_parser.add_argument("package", type=str)
 
-cli(root_parser.parse_args())
+cmd_remove_parser = root_subparsers.add_parser("new")
+cmd_remove_parser.add_argument("project_name", type=str)
```

Comment:
- `cmd_remove_parser` is meant to be used by remove command (`protostar remove <package_name>`). Each command should have its own subparser.
- I think `init` command would be a better name than `new` because it is a verb and is used more commonly ([foundry uses init](https://onbjerg.github.io/foundry-book/forge/creating-a-new-project.html)).